dsl/src/main/scala/org/http4s/dsl/Http4sDsl.scala

```diff
@@ -17,6 +17,7 @@ trait Http4sDsl[F[_]] extends Http4s with Methods with Statuses with Responses[F
   val :? : impl.:?.type = impl.:?
   val ~ : impl.~.type = impl.~
   val -> : impl.->.type = impl.->
+  val → : impl.->.type = impl.->
```

Comment:
I'd put a scaladoc on this to note the precedence issue.  It's obvious when you mention it, but I didn't think about it beforehand.